src/commonComponents/searchBox/SearchBox.js

```diff
@@ -0,0 +1,31 @@
+// Library
+import PropTypes from 'prop-types'
+
+// icon
+import SearchIcon from '../../assests/images/search.svg'
+
+// css
+import './searchBox.css'
+
+function SearchBox(props){
+
+    const { placeholder } = props;
+
+    return(
+        <div className={`inline filter-inputs search-box`}>
+            <img src={SearchIcon} alt="search" />
```

Comment:
1. take the icon as a prop and keep this icon as your defaultProp
2. also make sure to take className as a prop